solutions/usaco-856.mdx

```diff
@@ -39,6 +39,54 @@ file_out.close()
 ```
 
 </PySection>
+	
+<JavaSection>
+	
+```java
+import java.io.*;
+import java.util.*;
+
+public class TheBucketList {
```

Comment:
```suggestion
public class BList {
```

Reply:
thats not the problem name

Reply:
yes it is? that's the name of the input file?

Reply:
the name of the input file is not the name of the problem

Reply:
it's the "system" name of the problem
like in the training gate the task name is always something like that you know
> then at least just `BucketList`, The is just unnecessary

Reply:
i think BucketList or TheBucketList are fine, since imo we should use the actual name instead of the filename